### ISSUE TITLE
Fails on loading history

### ISSUE THREAD

**Issue by alex-kravets** [jupyterlab/jupyterlab-git#473]
##  Description

There is a bug that prevents the plugin from working smoothly. Actually, after pure installation I cannot work with the plugin at all. Plugin fails during loading repo history because of accessing an element in a dict by a key that does not exist.

## Reproduce

1. Create a new folder and initialize a git repo inside of it
2. Run jupyter lab from this folder
3. Go to Git panel on the left side by clicking the icon
4. Try to make a commit or look at commits history
5. See the note inside the bar telling you that git repo isn't recognized correctly

Second option:
1. Run jupyter lab from a new folder
2. Initialize a git repo with the usage of the plugin
3. Make a commit
4. Make one more commit
5. Look at in the history bar

## Expected behavior

You will see no history points there. And if you selected option 2 - you will get the error message with traceback inside the terminal session (where you've started a jupyter lab).

## Proposal

As I figured out, the error came from this code (handlers.py):
```
class GitAllHistoryHandler(GitHandler):

    def post(self):
        body = self.get_json_body()
        current_path = body["current_path"]
        history_count = body["history_count"]
```
Bug in the last line: during the execution there is no `history_count` key inside the body dict. So I propose to replace that line by the following:
```
history_count = body.get("history_count", 25)
```

PS. Thanks for a great tool for jupyter lab, really love it :)

**Comment by fcollonval** [comment]
@alex-kravets Thanks for reporting this bug. Could you provide the version of the python package and the jupyterlab extension you are using?

**Comment by alex-kravets** [comment]
@fcollonval Of course :)
I'm using Python 3.7.5 with the following packages:
jupyterlab==1.2.3
jupyterlab-git==0.9.0
jupyterlab-server==1.0.6

**Comment by telamonian** [comment]
> pure installation

What exactly do you mean by that? From your version numbers, it sounds like you're installing the `jupyterlab-git` plugin from a local copy of the repository. If so, that's important information for figuring out what's going wrong. You should also run `jupyter labextension list` and post the output here.

> So I propose to replace that line by the following:
> 
> ```
> history_count = body.get("history_count", 25)
> ```

You could solve your problem that way, but it wouldn't be a great idea. I think in this case you'd just end up hiding your real problem. Here's what I mean: a default value of `history_count` is already getting set by the frontend

https://github.com/jupyterlab/jupyterlab-git/blob/a656567517459ef1ef9b24d2f27fcd4260d230f7/src/model.ts#L295-L310

So if you also set a default value on the backend, you then end up hiding whatever bug is causing the `history_count` value to get screwed up in the first place.

I think there's two possibilities:
1. You have an old version of `jupyterlab-git`. We've fixed numerous bugs over the last couple of weeks, so you should try to install the latest version (we released `v0.9.0` on pypi just a few hours ago). Run:

```
pip install -U jupyterlab-git
jupyter lab build
```

2. Something about your installation of jupyterlab is messed up. Currently, the least painful way to reliably fix a jlab install is to first delete the entire contents of your jupyterlab app dir (you can find the path to it by running `jupyter lab paths`) and then run `jupyter lab build`. 

The crappy thing is that you'll then have to reinstall all of your extensions. You *might* be able to get around that by skipping over the `extensions` dir when you're cleaning out your app dir

**Comment by fcollonval** [comment]
Closing this as this was related to a installation problem: mismatch between the server and the frontend extension.
